apps/omg_child_chain/test/omg_child_chain/block_queue/gas_analyzer_test.exs

```diff
@@ -34,6 +34,7 @@ defmodule OMG.ChildChain.BlockQueue.GasAnalyzerTest do
       gas_analyzer: gas_analyzer,
       handler_id: handler_id
     } do
+      # this is our mock that we will use as a mock of ethereumex
```

Comment:
A little way around, rephrase needed: `this module serves as a mock of the ethereumex` ❓